posts/01-05-18/piping-hot-a-rundown-on-the-pipeline-operator.md

```diff
@@ -0,0 +1,134 @@
+# Piping Hot: A Rundown on the Pipeline Operator
+
+## Context
+The Pipeline operator is one of the more interesting proposals advancing through the TC39 currently.
+
+The operator `|>` is borrowed and inspired by several functional languages such as F#, Elm, and Elixir, and UNIX pipes as well. (The original syntax was `::`, which proved too controversial) According to the TC39 proposal
+
+> It's a backwards-compatible way of streamlining chained function calls in a readable, functional manner, and provides a practical alternative to extending built-in prototypes.
+
+which makes it very useful for short single-argument functions.
+
+## Examples
+
+In the simplest example, the pipeline operator turns `alert('Hello!')` into `'Hello!' |> alert`. This isn't a particularly good example of course, as it doesn't aid the readability in any way. The better usage comes when you chain several functions together.

```

Comment:
sounds like you are contradicting yourself. YOu can show this example later in the post